packages/components/input/src/InputClearButton.tsx

```diff
@@ -47,4 +47,8 @@ export const InputClearButton = forwardRef<HTMLButtonElement, InputClearButtonPr
   }
 )
 
+export const InputClearButton = Object.assign(Root, {
+  id: 'ClearButton',
+})
```

Comment:
I'm still not familiar with assigning an `id` to the component like this instead of a displayName. Can you summarize it please ? :)

Reply:
Yes. I did this change to not be coupled with the `displayName`. For example, to add the `Textarea` I would have needed to go to the `InputGroup` and add support for it
```tsx
const input = findElement('Input', 'Textarea')
```
Doing it this way no changes need to be performed in the `InputGroup` and consumer can also created their own component that can work nicely with the `InputGroup` is needed extending the `id`.

I was inspired by chakra [InputGroup](https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/input/src/input-group.tsx#L44) to perform this change